### PR TITLE
OSDOCS-5893:creates 4.14 RN template

### PR DESCRIPTION
**Pull request by JoeAldinger**
[OSDOCS#5983](https://issues.redhat.com//browse/OSDOCS-5893): updates RNs for 4.14

Version(s):
4.14

Issue:
https://issues.redhat.com/browse/OSDOCS-5983

Link to docs preview:
http://file.rdu.redhat.com/jaldinge/OSDOCS-5893-RN/release_notes/ocp-4-14-release-notes.html

QE review:
- [ ] QE has approved this change.

Additional information:
Changes in main https://github.com/openshift/openshift-docs/pull/60319

### CODE REVIEW

File: migrating_from_ocp_3_to_4/planning-migration-3-4.adoc [hunk 1]
```diff
@@ -9,7 +9,7 @@ toc::[]
 {product-title} {product-version} introduces architectural changes and enhancements/ The procedures that you used to manage your {product-title} 3 cluster might not apply to {product-title} 4.
 
 ifndef::openshift-origin[]
-For information on configuring your {product-title} 4 cluster, review the appropriate sections of the {product-title} documentation. For information on new features and other notable technical changes, review the xref:../release_notes/ocp-4-13-release-notes.adoc#ocp-4-13-release-notes[OpenShift Container Platform 4.13 release notes].
+For information on configuring your {product-title} 4 cluster, review the appropriate sections of the {product-title} documentation. For information on new features and other notable technical changes, review the xref:../release_notes/ocp-4-14-release-notes.adoc#ocp-4-14-release-notes[OpenShift Container Platform 4.13 release notes].
```
Comment (adellape) [inline migrating_from_ocp_3_to_4/planning-migration-3-4.adoc:12]:
```suggestion
For information on configuring your {product-title} 4 cluster, review the appropriate sections of the {product-title} documentation. For information on new features and other notable technical changes, review the xref:../release_notes/ocp-4-14-release-notes.adoc#ocp-4-14-release-notes[OpenShift Container Platform 4.14 release notes].
```

File: migrating_from_ocp_3_to_4/planning-migration-3-4.adoc [hunk 2]
```diff
@@ -21,7 +21,7 @@ With {product-title} 3, administrators individually deployed {op-system-base-ful
...
 
-Beginning with {product-title} 4.13, {op-system} now uses {op-system-base-full} 9.2 packages. This enhancement enables the latest fixes and features as well as the latest hardware support and driver updates. For more information about how this upgrade to RHEL 9.2 might affect your options configuration and services as well as driver and container support, see the xref:../release_notes/ocp-4-13-release-notes.adoc#ocp-4-13-rhcos-rhel-9-2-packages[RHCOS now uses RHEL 9.2] in the _OpenShift Container Platform 4.13 release notes_.
+Beginning with {product-title} 4.13, {op-system} now uses {op-system-base-full} 9.2 packages. This enhancement enables the latest fixes and features as well as the latest hardware support and driver updates. For more information about how this upgrade to RHEL 9.2 might affect your options configuration and services as well as driver and container support, see the xref:../release_notes/ocp-4-14-release-notes.adoc#ocp-4-14-rhcos-rhel-9-2-packages[RHCOS now uses RHEL 9.2] in the _OpenShift Container Platform 4.13 release notes_.
```
Comment (adellape) [inline migrating_from_ocp_3_to_4/planning-migration-3-4.adoc:24]: Mismatch on the file path and the _OpenShift Container Platform 4.13 release notes_, but should this remain as a link to the 4.13 relnotes specifically, or will this `*-rhcos-rhel-9-2-packages` content maybe be carried over into the 4.14 relnotes?

Comment (adellape) [reply migrating_from_ocp_3_to_4/planning-migration-3-4.adoc:24]: Or another option: maybe this para should just be removed from the 4.14+ docs. 🤷

Comment (jldohmann) [reply migrating_from_ocp_3_to_4/planning-migration-3-4.adoc:24]: I would just remove it, there is currently no commitment from engineering that RHCOS 4.14 should expect a RHEL bump

Comment (kalexand-rh) [reply migrating_from_ocp_3_to_4/planning-migration-3-4.adoc:24]: Perhaps we should keep it with the hard link. If people are going from EUS to EUS versions, they might have missed this change in 4.13.

Comment (ahardin-rh) [reply migrating_from_ocp_3_to_4/planning-migration-3-4.adoc:24]: I agree with @jldohmann. We can just remove it for now.

Comment (JoeAldinger) [reply migrating_from_ocp_3_to_4/planning-migration-3-4.adoc]: For now, I'm going to put the hardcoded link in and we can revisit this closer to GA. I could see a use case for the link in 4.12 to 4.14 updates.

**Pull request merged**